src/main/db/repository/publication.ts

```diff
@@ -212,7 +213,7 @@ export class PublicationRepository  /* extends BaseRepository<PublicationDocumen
     }
 
 
-    public async searchByTitle(title: string): Promise<PublicationDocument[]> {
+    public async searchByTitleAndAuthor(title: string): Promise<PublicationDocument[]> {
```

Comment:
Nit picking: could you please rename `title` to a more generic variable name like `str` or `keyword`?

Reply:
Fixed via https://github.com/edrlab/thorium-reader/pull/1551/commits/1c90a5b058f77b8e09a621fedbe3b09104679f01